src/main/kotlin/com/cognifide/gradle/aem/instance/InstanceSync.kt

```diff
@@ -484,6 +486,30 @@ class InstanceSync(aem: AemExtension, instance: Instance) : InstanceHttpClient(a
         return scripts.asSequence().map { evalGroovyScript(it, data, verbose) }
     }
 
+    override fun getNode(path: String): Node? {
```

Comment:
Use Kotlin delegation feature instead of this "java style proxy":
https://kotlinlang.org/docs/reference/delegation.html